samples/swiftui/FirebaseSwiftUIExample/FirebaseSwiftUIExampleUITests/FirebaseSwiftUIExampleUITests.swift

```diff
@@ -81,10 +81,16 @@ final class FirebaseSwiftUIExampleUITests: XCTestCase {
     signInButton.tap()
 
     let signedInText = app.staticTexts["signed-in-text"]
-    XCTAssertTrue(
-      signedInText.waitForExistence(timeout: 10),
-      "SignedInView should be visible after login"
-    )
+    let maxWaitTime: TimeInterval = 30
+    let interval: TimeInterval = 1
+    var waited: TimeInterval = 0
+
+    while !signedInText.exists && waited < maxWaitTime {
+      sleep(UInt32(interval))
+      waited += interval
+    }
```

Comment:
You should use [`waitForExpectations`](https://developer.apple.com/documentation/xctest/xctestcase/waitforexpectations(timeout:handler:)) instead.